tests/server-common/src/test/java/io/a2a/server/apps/common/AbstractA2AServerTest.java

```diff
@@ -9,39 +9,14 @@
 import static org.wildfly.common.Assert.assertNotNull;
 import static org.wildfly.common.Assert.assertTrue;
 
-import java.io.EOFException;
-import java.io.IOException;
-import java.net.URI;
-import java.net.http.HttpClient;
-import java.net.http.HttpRequest;
-import java.net.http.HttpResponse;
-import java.nio.charset.StandardCharsets;
-import java.util.List;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.concurrent.atomic.AtomicReference;
-import java.util.stream.Stream;
-
-import jakarta.ws.rs.core.MediaType;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-
 import io.a2a.client.A2AClient;
 import io.a2a.spec.A2AServerException;
 import io.a2a.spec.AgentCard;
 import io.a2a.spec.Artifact;
-import io.a2a.spec.CancelTaskRequest;
 import io.a2a.spec.CancelTaskResponse;
 import io.a2a.spec.DeleteTaskPushNotificationConfigResponse;
 import io.a2a.spec.Event;
-import io.a2a.spec.GetTaskPushNotificationConfigParams;
```

Comment:
@misselvexu Thanks for keeping at it! Just triggered CI and noticed there's a compilation failure. Looks like some imports have been removed but they are still needed below.